go-controller/pkg/cni/cni.go

```diff
@@ -138,16 +163,52 @@ func (pr *PodRequest) cmdAdd(kubeAuth *KubeAPIAuth, podLister corev1listers.PodL
 	return response, nil
 }
 
-func (pr *PodRequest) cmdDel() error {
-	if config.OvnKubeNode.Mode == types.NodeModeDPUHost {
-		// nothing to do
-		return nil
+func (pr *PodRequest) cmdDel(podLister corev1listers.PodLister, kclient kubernetes.Interface) (*Response, error) {
+	// assume success case, return an empty Result
+	response := &Response{}
+	response.Result = &current.Result{}
+
+	namespace := pr.PodNamespace
+	podName := pr.PodName
+	if namespace == "" || podName == "" {
```

Comment:
@girishmg doesn't this already get validated in `cniRequestToPodRequest()`?

Reply:
@dcbw you are right, we can drop this check. I will remove similar code from cmdDel as well.